src/org/jitsi/impl/neomedia/MediaServiceImpl.java

```diff
@@ -1531,9 +1532,17 @@ public static boolean isJmfRegistryDisableLoad()
      */
     private static void postInitializeOnce(MediaServiceImpl mediaServiceImpl)
     {
-        new ZrtpFortunaEntropyGatherer(
-                mediaServiceImpl.getDeviceConfiguration())
-            .setEntropy();
+        /*
+         * Some SecureRandom() implementations like SHA1PRNG call
+         * /dev/random to seed themselves on first use.
+         * Call SecureRandom early to avoid blocking when establishing
+         * a connection for exemple.
+         */
+        logger.info("Warming up SecureRandom...");
+        SecureRandom rnd = new SecureRandom();
+        byte[] b = new byte[20];
+        rnd.nextBytes(b);
+        logger.info("Warming up SecureRandom finished.");
```

Comment:
Object on stack doing lots of stuff using reflection, I can imagine a compiler seeing this as 'does nothing, throw away'. Anyway, javac doesn't do much optimization anyway and leaves to Hotspot. So I guess it should be safe enough.